File: src/main/java/com/bot/model/dto/DynamicsDataDto.java
package com.bot.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import org.jfree.data.time.Month;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class DynamicsDataDto {
    private Month month;
    private BigDecimal spend;
    private String userName;

    @Override
    public String toString() {
        return month.toString() + " - " + spend;
    }
}

File: telegram-adapter/src/main/kotlin/com/planner/model/dto/DynamicsDataDto.kt
package com.planner.model.dto

import org.jfree.data.time.Month
import java.math.BigDecimal

data class DynamicsDataDto(
    val month: Month,
    val spend: BigDecimal,
    val userName: String
) {
    override fun toString() = month.toString() + " - " + spend
}